app/assets/javascripts/datatables_defaults.js

```diff
@@ -3,6 +3,7 @@
 /* https://datatables.net/reference/option/ */
 
 $.extend( true, $.fn.dataTable.defaults, {
+  "dom": "fr<'event-table-info't>ip",
```

Comment:
This file is for all datatables, isn't it ? Seems odd to have something specific to events here.

Reply:
@kgf This creates a div with class event-table-info specifically for adding in elements in JS to the datatables. This is the original revision which I plan on creating another commit to change but wanted you verify the functionality first

Reply:
Functionality looks really nice